modules/howtos/examples/TransactionsExample.java

```diff
@@ -217,9 +240,24 @@ static void examplesReactive() {
                     })
 
                     // Getting and removing a doc:
-                    .then(ctx.get(collection.reactive(), "doc-c")).flatMap(doc -> ctx.remove(doc))
+                    .then(ctx.get(collection.reactive(), "doc-c"))
+                        .flatMap(doc -> ctx.remove(doc))
+
+                    // Performing a SELECT N1QL query against a scope:
+                    .then(ctx.query(inventory, "SELECT * FROM hotel WHERE country = $1",
+                            TransactionQueryOptions.queryOptions()
+                                    .parameters(JsonArray.from("United Kingdom"))))
+
+                    .doOnNext(queryResult -> queryResult.rowsAs(JsonObject.class)
+                            .doOnNext(row -> {
+                                // the application would do something with each row here
+                            }))
+
+                    // Performing an UPDATE N1QL query against a scope:
```

Comment:
might add to this comment "against a scope containing a collection named `hotel`" or something like this

Reply:
Will do